backend/apps/readings/analysis.py

```diff
@@ -3,9 +3,67 @@
 Analysis.py - analyses for dhmit/rereading wired into the webapp
 
 """
+from statistics import stdev
+from pathlib import Path
+
 from .models import StudentResponse
 
 
+def get_sentiments() -> dict:
+    """
+    Returns a dictionary of sentiment scores, with the keys being the word and the values being
+    their score
+
+    :return: dict mapping words to their sentiment scores
+    """
+    sentiment_path = Path('..', 'analysis', 'data', 'sentiments.txt')
+
+    sentiments = dict()
+    with open(sentiment_path, 'r') as file:
+        word = file.readline()
```

Comment:
Try not to depend on functions advancing the line itself.  Instead of `word = file.readline()` over again, use a for loop: `for word in file:`